src/test/unit-test/config.test.js

```diff
@@ -100,6 +100,20 @@ describe('Test Config class', function () {
         })
     });
 
+    describe('Test setCustomFieldsTypeConversion', function () {
+        beforeEach(function () {
+            singleton.setCustomFieldsTypeConversion("retain");
+            result = singleton.getConfig();
+        });
+        it('sets custom fields format', function () {
```

Comment:
if the test is called set custom fields format, than the changing of the type conversion should be done in the test and not in the before and after methods.

Reply:
Actually, the `it` is an essential part of the description:
"it sets custom fields format" is not referring to the test code but to the code under test.